### DISCUSSION TITLE
Minor point releases for essential bug fixes?

### DISCUSSION THREAD

**Discussion by StarTroop** [qtile/qtile#4611] [General]
With the recent v23 release being a year after the previous version, has there been any discussion about increasing the release rate? I personally prefer to use the official packages provided by my distribution whenever possible, but since they only follow upstream's stable releases, many bugs may be left open for a long period of time.  
Recently there was the #4417 fix which I consider essential to the v23 release. Now, I don't mind major versions are spread out to bank new features, but it would be nice to receive the fixes in a more timely manor without having to resort to dev builds.
Any thoughts?  
P.S. I greatly appreciate all your efforts, and I don't intend to pressure you into a more than reasonable level of work.

**Comment by elParaguayo** [comment]
Great timing. We've just been discussing on discord and all agree that we want to release much more regularly!

**Reply by StarTroop** [reply]
Thanks! I guess I should check out the Discord more often.
